core/codenvy-hosted-workspace-activity/src/main/java/com/codenvy/activity/server/WorkspaceActivityService.java

```diff
@@ -56,9 +56,6 @@ public WorkspaceActivityService(WorkspaceActivityManager workspaceActivityManage
     @Path("/{wsId}")
     public void active(@PathParam("wsId") String wsId) throws ForbiddenException, NotFoundException, ServerException {
```

Comment:
I don't know who is "developer plugin" and why he can't read javadoc and need swagger